HelloWorld/HelloWorld/Program.cs

```diff
@@ -10,8 +10,8 @@ internal class Program
 	{
 		private static void Main(string[] args)
 		{
-			Console.WriteLine("Hello World!");
+			Console.WriteLine("How are you going?");
```

Comment:
If we really want our application to take the world by storm, we should handle any language. A new translation class, GreetingsHelper, has been added to the Helpers library. Let's use that.